documentation/explainers/price-feed-identifiers.md

```diff
@@ -0,0 +1,77 @@
+# Price feed identifiers
+
+This document explains how price feed identifiers are configured and how they are used.
+
+## What is an identifier
+
+An identifier is the term we use to refer to an underlying asset whose price a derivative tracks. Examples of underlying assets
+are the price of Bitcoin in US dollars, the spot price of gold, or the daily high temperature in New York City.
+
+## Configuration file
+
+Price feed identifiers are configured in [identifiers.json](`https://github.com/UMAprotocol/protocol/blob/master/core/config/identifiers.json`).
+This json file contains a list of entries for each supported identifier, with two bits of configuration information:
+1. `dappConfig`: configuration for the Sponsor dApp.
+2. `uploaderConfig`: configuration for UMA's price feed uploader.
+See the sections below for details on these configurations.
+
+To support a new identifier, follow these steps:
+1. Add an entry to `identifiers.json` with both `dappConfig` and `uploaderConfig`.
+2. Approve the identifier in the Oracle. Locally, you can run:
+
+   ```
+   $(npm bin)/truffle exec scripts/local/ApproveIdentifiers.js --network=test
+   ```
+
+3. Push at least one price. Locally, you can manually push a price:
+
+   ```
+   $(npm bin)/truffle exec scripts/ManualPublishPriceFeed.js --identifier <identifier> --price <price> --time <time> --network=test
+   ```
+
+   Or you can run UMA's price feed uploader:
+
+   ```
+   $(npm bin)/truffle exec scripts/PublishPrices.js --network=test
+   ```
+
+   Note that running the price feed may require several API keys to be provided as environment variables.
+
+## Sponsor dApp
+
+The [Sponsor dApp](https://github.com/UMAprotocol/protocol/tree/master/sponsor-dapp-v2) references the `dappConfig` field.
+If the identifier is supported, the dApp allows you to select it as the index for your synthetic token.
+
+The `dappConfig` section contains two fields:
+1. `expiries`: timestamps of expirations supported in the dApp for synthetic tokens tracking this index. A value of `0`
+   indicates no expiry (i.e., a perpetual product). Expiries are configured per identifier.
+2. `supportedMove`: fraction of move in the underlying price. Higher volatility assets should have a higher
+   `supportedMove`. Financial contract templates use this value to reduce the probability of liquidation by making sure
+   they hold on to enough collateral to support a move of this size.
+
+If you want to use a different expiry or a different supported move, you can configure your token via the command line.
```

Comment:
I'm a little confused at this line - can't you change the offered expiries via the `expiries` field above?

Reply:
Clarified a bit.